app/src/test/java/org/oppia/app/profile/ProfileChooserFragmentLocalTest.kt

```diff
@@ -0,0 +1,134 @@
+import org.oppia.domain.classify.InteractionsModule
+import org.oppia.domain.classify.rules.continueinteraction.ContinueModule
+import org.oppia.domain.classify.rules.dragAndDropSortInput.DragDropSortInputModule
+import org.oppia.domain.classify.rules.fractioninput.FractionInputModule
+import org.oppia.domain.classify.rules.itemselectioninput.ItemSelectionInputModule
+import org.oppia.domain.classify.rules.multiplechoiceinput.MultipleChoiceInputModule
+import org.oppia.domain.classify.rules.numberwithunits.NumberWithUnitsRuleModule
+import org.oppia.domain.classify.rules.numericinput.NumericInputRuleModule
+import org.oppia.domain.classify.rules.textinput.TextInputRuleModule
+import org.oppia.domain.question.QuestionModule
+import org.oppia.testing.FakeEventLogger
+import org.oppia.testing.TestDispatcherModule
+import org.oppia.testing.TestLogReportingModule
+import org.oppia.util.caching.CacheAssetsLocally
+import org.oppia.util.gcsresource.GcsResourceModule
+import org.oppia.util.logging.LoggerModule
+import org.oppia.util.parser.GlideImageLoaderModule
+import org.oppia.util.parser.HtmlParserEntityTypeModule
+import org.oppia.util.parser.ImageParsingModule
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.LooperMode
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@RunWith(AndroidJUnit4::class)
+@LooperMode(LooperMode.Mode.PAUSED)
+@Config(
+  application = ProfileChooserFragmentLocalTest.TestApplication::class,
+  qualifiers = "port-xxhdpi"
+)
+class ProfileChooserFragmentLocalTest {
+
+  @Inject
+  lateinit var fakeEventLogger: FakeEventLogger
+
+  @Before
+  fun setUp() {
+    Intents.init()
+    setUpTestApplicationComponent()
+  }
+
+  @After
+  fun tearDown() {
+    Intents.release()
+  }
+
+  @Test
+  fun testProfileChooser_onLaunch_logsEvent() {
+    launch<ProfileActivity>(createProfileActivityIntent()).use {
+      val event = fakeEventLogger.getMostRecentEvent()
+
+      assertThat(event.priority).isEqualTo(Priority.ESSENTIAL)
+      assertThat(event.actionName).isEqualTo(EventAction.OPEN_PROFILE)
+      assertThat(event.context.activityContextCase).isEqualTo(ACTIVITYCONTEXT_NOT_SET)
+    }
+  }
+
+  private fun createProfileActivityIntent(): Intent {
+    return ProfileActivity.createProfileActivity(ApplicationProvider.getApplicationContext())
+  }
+
+  private fun setUpTestApplicationComponent() {
+    ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
+  }
+
+  @Module
+  class TestModule {
+    // Do not use caching to ensure URLs are always used as the main data source when loading audio.
+    @Provides
+    @CacheAssetsLocally
+    fun provideCacheAssetsLocally(): Boolean = false
+  }
+
+  // TODO(#59): Figure out a way to reuse modules instead of needing to re-declare them.
+  @Singleton
+  @Component(
+    modules = [
+      TestModule::class, TestDispatcherModule::class, ApplicationModule::class, NetworkModule::class,
```

Comment:
This line is going out of 100 char per line limit, how about if we put 3-3 or 2-2 class in a line, it will bring some consistency and looks good I think.

Reply:
Yes, I think 3-3 will be better. Done :)